README.md

```diff
@@ -170,12 +170,14 @@ A SuperAgent-driven library for testing HTTP servers, allowing you to test your
 | DELETE | `/api/products/:slug`    | Delete a specific product by slug         | Private  |
 | GET    | `/products/new-arrivals` | Get a list of new arrival products        | Public   |
 | GET    | `/products/feature`      | Get a list of featured products           | Public   |
+| GET    | `/products/relative-products/:categorySlug`      | Get related products           | Public   |
+| GET    | `/products/search?k=abc&c=def`      | Get products by keyword and categorySlug           | Public   |
 
 ### Categories
 
 | Method | Routes                  | Functionalities                     | Access  |
 | ------ | ----------------------- | ----------------------------------- | --------|
-| GET    | `/api/categories`       | Get a list of all categories        | Public  |
+| GET    | `/api/categories`       | Get a list of all categories        | Private |

```

Comment:
I missed this part from my last approval. The access should remain Public as it is.